AppKit/CPSplitView.j

```diff
@@ -1205,6 +1186,32 @@ The sum of the views and the sum of the dividers should be equal to the size of
 
 @end
 
+@implementation CPSplitView (CPTrackingArea)
+
+- (void)updateTrackingAreas
+{
+    var myTrackingAreas = [self trackingAreas];
+    
+    for (var i = 0; i < myTrackingAreas.length; i++)
+        [self removeTrackingArea:myTrackingAreas[i]];
```

Comment:
Actually, such a method would be "protected" in languages that support method visibility; i.e. it's visible to implementers of CPView subclasses but is not part of the public API. So I'd like to call it _removeAllTrackingAreas, but that means the documentation generator will ignore it, so it will have to be removeAllTrackingAreas. But be sure to document that the method is only for use by subclasses that want to override updateTrackingAreas.